frontend/app/config/webpack.config.js

```diff
@@ -421,7 +406,7 @@ module.exports = function (webpackEnv) {
             // The preset includes JSX, Flow, TypeScript, and some ESnext features.
             {
               test: /\.(js|mjs|jsx|ts|tsx)$/,
-              include: paths.appSrc,
+              include: [paths.appSrc, paths.libSrc],
```

Comment:
We test these inside `app` and `lib` to support both cases (essentially avoiding the issue from removing the ModuleScopePlugin).